common_features.mk

```diff
@@ -228,6 +229,10 @@ ifeq ($(strip $(CIE1931_CURVE)), yes)
     LED_TABLES = yes
 endif
 
+ifeq ($(strip $(FASTLED_LIB8TION)), yes)
+    SRC += $(LIB_PATH)/lib8tion/trig8.c
```

Comment:
Change this to `QUANTUM_LIB_SRC += $(LIB_PATH)/lib8tion/lib8tion.c`

Using QUANTUM_LIB_SRC means the linker won't include anything from this in the final binary unless they're actually referenced in the code. This is why it's unnecessary to make it conditional above.